Sources/MessagingInApp/Gist/Managers/ModalViewManager.swift

```diff
@@ -40,13 +43,15 @@ class ModalViewManager {
             finalPosition = viewController.view.center.y - viewController.view.bounds.height
         }
 
-        UIView.animate(withDuration: 0.3, delay: 0, options: [.curveEaseIn], animations: {
+        animationRunner.animate(withDuration: 0.3, delay: 0, options: [.curveEaseIn], animations: {
```

Comment:
more of an SDK change so happy to not include but would it make more sense to use `weak self`  over here?  incase If the ModalViewManager is deallocated during an ongoing animation, the animation will simply stop without causing a crash.